dbt_schema_builder/relation.py

```diff
@@ -182,3 +192,59 @@ def _dir_is_flat(self, path):
             if entry.is_dir():
                 return False
         return True
+
+    @staticmethod
```

Comment:
Not super important, but does it make more sense to write this as a normal method, seeing that this object already has an app and redactions attribute?

Reply:
Agreed. I had done some of that kind of thing, but just ran out of time for this PR. I had to roll it out since a number of people are waiting on this to merge.